tests/render/fixtures/ValidationResultsPageRenderer_render_with_run_info_at_end.json

```diff
@@ -651,14 +651,17 @@
                 {
                   "content_block_type": "string_template",
                   "string_template": {
-                    "template": "values must belong to this set: [ ].",
+                    "template": "values must belong to this set: $v__0 $v__1 $v__2.",
```

Comment:
Does the `$v__*` syntax have any semantic meaning? That is, why 2 underscores?

Reply:
it's an templating syntax (either one of our libs or internal I don't remember)